### ISSUE TITLE
on/ off option on controllers in controller-rack

### ISSUE THREAD

**Issue by musikBear** [LMMS/lmms#3686]
Currently there is no automateable option for toggling of a controller in controller-rack.
I suggest something like
![onoffonrack](https://user-images.githubusercontent.com/6662066/27924160-a0916164-6280-11e7-9ad9-259261a5cf34.JPG)
Where contoller1 is active and controller2 is disabled. Those settings should be automateable.

**Comment by zonkmachine** [comment]
Related or duplicate of https://github.com/LMMS/lmms/issues/1997

**Comment by tresf** [comment]
Related but slightly different (and actually better). #1997 puts the automation on the specific assignment.  I feel it would make much more sense to put it on the controller itself rather than disconnecting and reconnecting all the time.

Note, a workaround for now is to use an automation event which zeros out the "AMT" value.

![image](https://user-images.githubusercontent.com/6345473/27934076-1e44dca8-6272-11e7-8104-2bce76fe4571.png)

**Comment by musikBear** [comment]
> Note, a workaround for now is to use an automation event which zeros out the "AMT" value.

Yes, i use that, but the 'thing' is that there then is a dial value that need to be recreated. Easy if user knows the program in depth, but not intuitive.

**Comment by BaraMGB** [comment]
Want just bring in this pr of mine. https://github.com/LMMS/lmms/pull/2944

But this is relevant.

**Comment by gi0e5b06** [comment]
Partly done. Internally the controller is still running, which may or may not make sense. However, a constant value (the last one) is returned. Would probably be better to improve the original model but that would require deeper changes.
![enable_controllers_20170722](https://user-images.githubusercontent.com/30031949/28492858-1bb67924-6f0b-11e7-9711-bd57f339a703.png)
